features/admin/quota.feature

```diff
@@ -2,7 +2,7 @@ Feature: Quota related scenarios
 
   # @author qwang@redhat.com
   @admin
-  @4.12 @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
+  @4.12 @4.11
```

Comment:
Is the change here expected ?

Reply:
Hi Liang, yes, as in the last comment,we do not plan to run the case on lower versoin of ocp.

Reply:
This is a different test case than the one we talked last time.  Please check carefully.

Reply:
Thanks for cactching up the mistake, it's updated now.